<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WritePoem</class>
 <widget class="QDialog" name="WritePoem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World of Poetry - Write Poem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
 color: rgb(0, 32, 63);


background-image: url(:/img/images/Background.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Write a Poem</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_poemTitle">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_poemTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_category">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_category">
       <property name="styleSheet">
        <string notr="true">
 color: rgb(0, 32, 63);
background-color: rgb(230, 222, 204);</string>
       </property>
       <item>
        <property name="text">
         <string>Love</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Life</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Happiness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sadness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Friendship</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_content">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_poemContent">
     <property name="placeholderText">
      <string>Express your feelings through poetry...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_submit">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
